maze/README.rst

```diff
@@ -0,0 +1,78 @@
+Maze Solver Demo
+================
+Getting the D-Wave Quantum Computer to solve a maze!
+
+The purpose of this demo is to take a simple and familiar problem - solving a maze - and go through
+the steps of submitting the problem to the quantum computer.
+
```

Comment:
The following code takes a simple and familiar problem---solving a maze---and demonstrates
the steps of submitting such problems to the quantum computer.